import-export-cli/integration/testutils/policy_testUtils.go

```diff
@@ -0,0 +1,329 @@
+	assert.Contains(t, output, "Successfully updated")
+	assert.Nil(t, err, "Error while importing the Throttling Policy")
+	// Give time for newly imported Throttling Policy to get indexed
+	base.WaitForIndexing()
+
+	// Get Throttle Policy from env 2
+	importedPolicy, _ := getThrottlingPolicyByName(t, args, adminUsername, adminPassword, policyName, policyType)
+	// Validate env 1 and env 2 policy is equal
+	validatePoliciesEqual(t, args, importedPolicy)
+	removeExportedThrottlingPolicyFile(t, args.ImportFilePath)
+}
+
+// ValidateThrottlePolicyImportUpdateConflict : Validates importing existing Throttling Policy to create conflict
+func ValidateThrottlePolicyImportUpdateConflict(t *testing.T, args *PolicyImportExportTestArgs, adminUsername, adminPassword, policyType string) {
+	const conflictStatusCode = "409"
+	t.Helper()
+
+	// Setup apictl envs
+	base.SetupEnv(t, args.DestAPIM.GetEnvName(), args.DestAPIM.GetApimURL(), args.DestAPIM.GetTokenURL())
+
+	// Export policy from env 1
+	base.Login(t, args.DestAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+	policyName := fmt.Sprintf("%v", args.Policy[policyNameKey])
+
+	exportedOutput, _ := exportThrottlePolicy(t, policyName, args.DestAPIM.GetEnvName(), args.Type)
+	args.ImportFilePath = base.GetExportedPathFromOutput(exportedOutput)
+	assert.True(t, base.IsFileAvailable(t, args.ImportFilePath))
+
+	// Import Throttling Policy to env 2
+	base.Login(t, args.DestAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+	output, err := importThrottlePolicy(t, args)
+	assert.Error(t, err, "Importation conflict expected")
+	assert.Contains(t, output, conflictStatusCode, "Unexpected error code")
+	// Give time for newly imported Throttling Policy to get indexed
+	base.WaitForIndexing()
+
+	// Get Throttle Policy from env 2
+	importedPolicy, _ := getThrottlingPolicyByName(t, args, adminUsername, adminPassword, policyName, policyType)
+	// Validate env 1 and env 2 policy is equal
+	validatePoliciesEqual(t, args, importedPolicy)
+	removeExportedThrottlingPolicyFile(t, args.ImportFilePath)
+}
+
+func ValidateThrottlePolicyImportFailureWithCorruptedFile(t *testing.T, args *PolicyImportExportTestArgs) {
+	const internalServerError = "500"
+	t.Helper()
+
+	// Setup apictl envs
+	base.SetupEnv(t, args.DestAPIM.GetEnvName(), args.DestAPIM.GetApimURL(), args.DestAPIM.GetTokenURL())
+
+	// Export policy from env 1
+	base.Login(t, args.DestAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+
+	path, _, _ := createExportedThrottlePolicyFile(t, args.DestAPIM, apim.AdvancedThrottlePolicyType, true)
+	args.ImportFilePath = path
+	assert.True(t, base.IsFileAvailable(t, args.ImportFilePath))
+
+	//// Import Throttling Policy to dest
+	output, err := importThrottlePolicy(t, args)
+	assert.Error(t, err, "Importation failure expected")
+	assert.Contains(t, output, internalServerError, "Unexpected error code")
+	removeExportedThrottlingPolicyFile(t, args.ImportFilePath)
```

Comment:
Can't we put this functionality inside a t.cleanUp function in createExportedThrottlePolicyFile. So that once the test is over, the file will get automatically removed?